scripts/acton_lake.json

```diff
@@ -0,0 +1,106 @@
+{
+    "citation": "Michael J Vanni, Maria J Gonzalez, and William H Renwick. 2019. Long term limnological measures in Acton Lake, a southwest Ohio reservoir, and its inflow streams: 1992-2017. Environmental Data Initiative.",
```

Comment:
change the description to `Long-term data collected from Acton Lake and its inflow streams, on a suite of physical, chemical and biological variables investigating how Acton Lake, responds to changes in ecosystem subsidies of detritus (sediments) and nutrients.`